db/R/query.format.vars.R

```diff
@@ -112,6 +112,22 @@ query.format.vars <- function(bety,input.id=NA,format.id=NA,var.ids=NA){
                    lat = site.lat,
                    lon = site.lon
     )
+    
+    # Check that all bety units are convertible. If not, throw a warning. 
+    for(i in 1:length(format$vars$bety_units)){
+      
+      if( format$vars$storage_type[i] != ""){ #units with storage type are a special case
+        
+        # This would be a good place to put a test for valid sotrage types. Currently not implemented. 
+        
+      }else if(udunits2::ud.are.convertible(format$vars$input_units[i], format$vars$pecan_units[i]) == FALSE){ 
+        
+        if(misc.are.convertible(format$vars$input_units[i], format$vars$pecan_units[i]) == FALSE){
```

Comment:
@mdietze I thought this bit of code was for checking whether `input_units` and `pecan_units` are convertible when they are different, and do nothing when they're the same given these warnings:

```
2017-06-01 14:05:20 WARN   [query.format.vars] : 
   Units not convertible for CO2 with units of ppm (= umol mol-1) .  Please 
   make sure the varible has units that can be converted to ppm (= umol 
   mol-1) 
2017-06-01 14:05:20 WARN   [query.format.vars] : 
   Units not convertible for DOY with units of DOY .  Please make sure the 
   varible has units that can be converted to DOY 

```
It doesn't give such warnings for most of the other variables because for units like `cm, celcius`
```
ud.are.convertible("cm", "cm")
ud.are.convertible("celsius", "celsius")
....
```
etc. `ud.are.convertible` returns `TRUE` and passes

if checking whether the units are recognized by udunits was (also) the intended behaviour, then that's fine